stko/calculators/macromodel_calculators.py

```diff
@@ -0,0 +1,250 @@
+            minimum_gradient=0.05
+        )
+        Calculator.__init__(self)
+
+
+class MacroModellForceFieldEnergy(MacroModelCalculator):
+    """
+    Uses MacroModel software to calculate energies.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        import stk
+        import stko
+
+        # Create a molecule whose energy
+        we want to know
+        mol = stk.BuildingBlock('NCCN')
+
+        # Create energy calculator
+        opls_ff = stko.MacroModellForceFieldEnergy(#Insert MacroModel
+        path here)
+
+        # Calculate the energy
+        results = opls_ff.get_results(mol)
+        energy = opls_ff.get_energy(mol)
+        unit_string = results.get_unit_string()
+
+    """
+
+    def __init__(self,
+                 macromodel_path,
+                 output_dir=None,
+                 timeout=None,
+                 force_field=16,
+                 ):
+        """
+        Initialize a :class:`.MacroModellForceFieldEnergy` instance.
+
+        Parameters
+        ----------
+        macromodel_path : :class:`str`
+            The full path to the MacroModel executable.
+
+        output_dir : :class:`str`
+            The path to the output directory.
+
+        timeout : :class:`int`
+            The number of seconds to wait for the calculation to
+            complete.
+
+        forcefield : :class:`int`
+            The force field to use.
+            Force field arguments can be the following:
+            +------------+------------+
+            |  1  | MM2               |
+            +------------+------------+
+            |  2  | MM3               |
+            +------------+------------+
+            |  3  | AMBER             |
+            +------------+------------+
+            |  4  | AMBER94           |
+            +------------+------------+
+            |  5  | OPLSA             |
+            +------------+------------+
+            |  10 | MMFF94 and MMFF94s|
+            +------------+------------+
+            |  14 | OPLS_2005         |
+            +------------+------------+
+            |  16 | OPLS3e            |
+            +------------+------------+
+
+        """
+        super().__init__(
+            macromodel_path=macromodel_path, output_dir=output_dir,
+            timeout=timeout,
+            force_field=force_field,
+        )
+        self._macromodel_path = macromodel_path
+        self._output_dir = output_dir
+        self._timeout = timeout
+        self._forcefield = force_field
+
+    def get_results(self, mol):
+        """
+        Calculate the energy of `mol`.
+
+        Parameters
+        ----------
+        mol : :class:`.Molecule`
+            The molecule to calculate the energy of.
+
+        Returns
+        -------
+        :class:`.EnergyResults`
+            The energy and units of the energy.
+
+        """
+
+        return EnergyResults(
+            generator=self.calculate(mol),
+            unit_string='kJ mol-1'
```

Comment:
Is it actually in kj/mol? That actually makes me like schrodinger more!! (I believe you, am just flabbergasted)